packages/compass-explain-plan/src/components/explain-stage/explain-stage.jsx

```diff
@@ -227,56 +228,51 @@ class ExplainStage extends Component {
     return (
       <div
         data-testid="explain-stage"
-        className={classnames(styles['explain-stage'])}
+        className={styles['explain-stage']}
         style={{
           zIndex: this.state.detailsOpen ? this.getNewZIndex() : 'initial',
           top,
           left,
         }}
       >
-        <h3 className={classnames(styles['stage-header'])}>
+        <Subtitle className={styles['stage-header']}>
           {this.props.name}
-        </h3>
-        <ul className={classnames(styles.core)}>
```

Comment:
Just for my own understanding, why are the `classnames()` calls only removed in some places?

Reply:
I just noticed they were redundant. Essentially all it does is `args.join(' ')`. So I just kept the ones where it actually joined stuff. For a moment there I got excited, thinking maybe I can remove the dep :) It usually goes away in these PRs.

Reply:
(So yes the changes are out of scope)